component.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "bacon.matchers",
-  "version": "0.4.1",
+  "version": "0.4.2",
```

Comment:
`bower.json` and `package.json` version fields are automatically updated by the release script when a new version is released, so updating it is not necessary. But no harm in doing so either. :)